internal/providers/terraform/aws/dynamodb_table.go

```diff
@@ -40,6 +40,13 @@ func NewDynamoDBTableResource(d *schema.ResourceData) schema.CoreResource {
 		targets = append(targets, newAppAutoscalingTarget(ref, ref.UsageData))
 	}
 
+	pitr := &aws.PointInTimeRecovery{Enabled: false}
+	if d.Get("point_in_time_recovery").Exists() {
```

Comment:
Would prefer we just use a boolean flag here rather than a separate type, no real need to. Just add a property on the dynamo db type called `PointInTimeRecoveryEnabled`

Reply:
Changed to a straight bool flag